Integration tests upgrading Java lang to version

[Bumping Java 17](https://github.com/nextflow-io/nextflow/commit/6e2ca4544800aca4d54a133cdcef7ced7d5a8554) in the gradle build causes some [tests to fail](https://github.com/nextflow-io/nextflow/actions/runs/3440630762) with the following exception:  


```
Nov-10 21:49:49.386 [Task submitter] ERROR nextflow.processor.TaskProcessor - Error executing process > 'RNASEQ:INDEX (transcript)'

Caused by:
  'java.lang.String java.lang.String.stripIndent()'

org.codehaus.groovy.runtime.InvokerInvocationException: java.lang.NoSuchMethodError: 'java.lang.String java.lang.String.stripIndent()'
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:113)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at org.codehaus.groovy.runtime.InvokerHelper.invokePogoMethod(InvokerHelper.java:1029)
	at org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1012)
	at org.codehaus.groovy.runtime.InvokerHelper.invokeMethodSafe(InvokerHelper.java:101)
	at nextflow.file.FileSystemPathFactory$_bashLib_closure3.doCall(FileSystemPathFactory.groovy:85)
	at nextflow.file.FileSystemPathFactory$_bashLib_closure3.call(FileSystemPathFactory.groovy)
	at nextflow.file.FileSystemPathFactory.lookup0(FileSystemPathFactory.groovy:105)
	at nextflow.file.FileSystemPathFactory.bashLib(FileSystemPathFactory.groovy:85)
	at nextflow.executor.SimpleFileCopyStrategy.getBeforeStartScript(SimpleFileCopyStrategy.groovy:354)
	at nextflow.executor.BashWrapperBuilder.getHelpersScript(BashWrapperBuilder.groovy:352)
	at nextflow.executor.BashWrapperBuilder.makeBinding(BashWrapperBuilder.groovy:214)
	at nextflow.executor.BashWrapperBuilder.buildNew0(BashWrapperBuilder.groovy:309)
	at nextflow.executor.BashWrapperBuilder.buildNew0(BashWrapperBuilder.groovy:154)
	at nextflow.executor.BashWrapperBuilder.build(BashWrapperBuilder.groovy:324)
	at nextflow.cloud.google.batch.GoogleBatchTaskHandler.submit(GoogleBatchTaskHandler.groovy:103)
	at nextflow.processor.TaskPollingMonitor.submit(TaskPollingMonitor.groovy:196)
	at nextflow.processor.TaskPollingMonitor.submitPendingTasks(TaskPollingMonitor.groovy:562)
	at nextflow.processor.TaskPollingMonitor.submitLoop(TaskPollingMonitor.groovy:387)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethodClosure(MetaClassImpl.java:1048)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1142)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at groovy.lang.Closure.call(Closure.java:406)
	at groovy.lang.Closure.run(Closure.java:493)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.NoSuchMethodError: 'java.lang.String java.lang.String.stripIndent()'
	at nextflow.cloud.google.util.GsBashLib.makeLib(GsBashLib.groovy:57)
	at nextflow.cloud.google.util.GsBashLib.render(GsBashLib.groovy:92)
	at nextflow.cloud.google.util.GsBashLib.memoizedMethodPriv$fromSessionSession(GsBashLib.groovy:112)
	at nextflow.cloud.google.util.GsBashLib.access$1(GsBashLib.groovy)
	at nextflow.cloud.google.util.GsBashLib$__clinit__closure2.doCall(GsBashLib.groovy)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:274)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at groovy.lang.Closure.call(Closure.java:412)
	at org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.lambda$call$0(Memoize.java:137)
	at org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:137)
	at org.codehaus.groovy.runtime.memoize.ConcurrentCommonCache.getAndPut(ConcurrentCommonCache.java:113)
	at org.codehaus.groovy.runtime.memoize.Memoize$MemoizeFunction.call(Memoize.java:136)
	at groovy.lang.Closure.call(Closure.java:428)
	at nextflow.cloud.google.util.GsBashLib.fromSession(GsBashLib.groovy)
	at nextflow.cloud.google.util.GsPathFactory.getBashLib(GsPathFactory.groovy:78)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	... 34 common frames omitted
```

--- a/plugins/nf-google/src/main/nextflow/cloud/google/util/GsBashLib.groovy
+++ b/plugins/nf-google/src/main/nextflow/cloud/google/util/GsBashLib.groovy
@@ -54,7 +54,7 @@ class GsBashLib extends BashFunLib<GsBashLib> {
     }
 
     protected makeLib() {
-        def ret = '''
+        '''
         nxf_gs_download() {
             local source=$1
             local target=$2

--- a/plugins/nf-google/src/main/nextflow/cloud/google/util/GsBashLib.groovy
+++ b/plugins/nf-google/src/main/nextflow/cloud/google/util/GsBashLib.groovy
@@ -54,7 +54,7 @@ class GsBashLib extends BashFunLib<GsBashLib> {
     }
 
     protected makeLib() {
-        '''
+        def ret = '''
         nxf_gs_download() {
             local source=$1
             local target=$2
@@ -84,7 +84,8 @@ class GsBashLib extends BashFunLib<GsBashLib> {
             local target=$2
             gsutil ${gs_opts[@]} cp -R "$name" "$target/$name"
         }
-        '''.stripIndent()
+        '''
+        "$ret".stripIndent()
     }
 
     @Override

--- a/plugins/nf-google/src/main/nextflow/cloud/google/util/GsBashLib.groovy
+++ b/plugins/nf-google/src/main/nextflow/cloud/google/util/GsBashLib.groovy
@@ -84,8 +84,7 @@ class GsBashLib extends BashFunLib<GsBashLib> {
             local target=$2
             gsutil ${gs_opts[@]} cp -R "$name" "$target/$name"
         }
-        '''
-        "$ret".stripIndent()
+        '''.stripIndent(true)
     }
 
     @Override

